[IMP] accounting: update pay by checks (#18381)
odoo/documentation: content/applications/finance/accounting/payments/pay_checks.rst

Reviewer: I would update this old intro while you are there. It reads too much like a flow, and we should instead just quickly present the feature, something like:

"When paying a bill in Odoo, you can select checks as a payment method and print them."

Reviewer: ```suggestion
To activate the checks payment method, go to :menuselection:`Accounting --> Configuration -->
Settings`, scroll down to the :guilabel:`Vendor Payments` section, enable :guilabel:`Checks`, and :guilabel:`Save`
```
We could simplify/shorten this.

Then, in another paragraph, you should introduce the check layout/print options:

- Check Layout
- Multi-Pages Check Stub
- Check Top Margin
- Check Left Margin

Pay by checks
=============

> When paying a bill, checks can be selected as a payment method and can be printed. Then the checks

Reviewer: ```suggestion
When paying a vendor bill, checks can be selected as a payment method and can be printed. Then the checks
```

and bank statements can be reconciled.

Configuration
=============

To activate the checks payment method, go to :menuselection:`Accounting --> Configuration -->
Settings`, and scroll down to the :guilabel:`Vendor Payments` section.

> After enabling :guilabel:`Checks`, different fields are available for check configuration, such as:

Reviewer: ```suggestion
After enabling :guilabel:`Checks`, different configuration options are available:
```
To simplify

> After enabling :guilabel:`Checks`, different fields are available for check configuration, such as:

Reviewer: ```suggestion
Settings`, scroll down to the :guilabel:`Vendor Payments` section, and enable :guilabel:`Checks`. Once enabled, several configuration options are available:
```
- The first sentence should mention enabling "checks", it's clearer that way.
- I also tried to simplify the second sentence.

> - :guilabel:`Check Layout`: select the check's layout to be used for printing.

Reviewer: ```suggestion
- :guilabel:`Check Layout`: select the layout to use for printing.
```
To simplify/avoid repetitions

> - :guilabel:`Check Layout`: select the check's layout to be used for printing.

Reviewer: ```suggestion
- :guilabel:`Check Layout`: select the layout to use for printing.
```
Suggestion to simplify

- :guilabel:`Multi-Pages Check Stub`: determine whether the check stub can span multiple pages
  when the number of invoices or payments associated with a single check exceeds the space available

>   on one page. When enabled, additional pages are automatically generated as needed.

Reviewer: ```suggestion
- :guilabel:`Multi-Pages Check Stub`: enable to let the check stub span multiple pages when the
  number of invoices or payments associated with a single check exceeds the space available on one
  page.
```
Just a few suggestions to clarify/simplify

- :guilabel:`Check Top Margin`: define the top margin used when printing checks.
- :guilabel:`Check Left Margin`: define the left margin used when printing checks.

.. note::
   - Some countries require specific modules to print checks; such modules may be installed by

>      default.

Reviewer: ```suggestion
```
See my suggestion above

   - Once the :guilabel:`Checks` setting is activated, the **Checks** payment method is

>      automatically set up in the :guilabel:`Outgoing Payments` tabs of **bank** journals.

Reviewer: ```suggestion
   - Once activated, the :guilabel:`Checks` payment method is automatically set up in the
     :guilabel:`Outgoing Payments` tabs of bank journals.
```

   - Some localizations have blank paper formats. They can be used to print the information of the
     check ad-hoc when needed. This requires the use of both :abbr:`MICR (Magnetic Ink Character Recognition)`
     ink or toner complying with the standards for check printing, as well as `check-quality paper
     <https://checkdepot.net/collections/blank-check-paper/products/top-format-blank-check-paper-cdt164>`_.
     Other information, such as the company name, bank account, and check number, is printed when
     creating the blank check.
   - Pre-printed check formats (non-blank checks) require pre-printed paper from a third-party
     vendor.

.. _accounting/pay-checks/pay-bill-check:

> Pay a supplier bill with a check

Reviewer: Could you also revise this section while we are there? It seems outdated/not so well-written :)

Configuration
=============

.. _accounting/pay-checks/activate-methods:

Activate checks payment methods

> -------------------------------

Reviewer: ```suggestion
```
I know you didn't add this heading, but it's not necessary since there is only one configuration step/heading.

   - Once the :guilabel:`Checks` setting is activated, the **Checks** payment method is
     automatically set up in the :guilabel:`Outgoing Payments` tabs of **bank** journals.

>    - Some countries require specific modules to print checks; such modules may be installed by

Reviewer: ```suggestion
```
Move that info above

     default. For instance, the :ref:`US Checks Layout <l10n_us/optional-modules>` module is

>      required to :ref:`print US checks <l10n_us/writing-checks>`.

Reviewer: ```suggestion
```
The example isn't necessary.

   - Pre-printed check formats (non-blank checks) require pre-printed paper from a third-party
     vendor.
   - Use one of the blank check formats to print the information of the check ad-hoc when needed.
     This requires the use of both :abbr:`MICR (Magnetic Ink Character Recognition)` ink or toner
     complying with the standards for check printing. Other information, such as the company name,

>      bank account, and check number, is printed when creating the blank check.

Reviewer: Did you confirm with a PO/expert that this applies to all localizations using checks?
